src/hash_table.c

```diff
@@ -12,7 +12,7 @@ hash_table make_table__(size_t bucket_size) {
   return new_table;
 }
 /* Creates a new empty hash table */
-hash_table make_table() { return make_table__(2); }
+hash_table make_table() { return make_table__(2000); }
 
```

Comment:
something is broken in hash_realloc raises the default size so it dosn't realloc before I figure out what's broken